### ISSUE TITLE
Batch normalization?

### ISSUE THREAD

**Issue by ldenoue** [TianzhongSong/Person-Segmentation-Keras#3]
I noticed the trainer uses train_batch_size = 16

In that case wouldn’t it be better to add a batch norm layer in each of the conv layers?

Laurent

**Comment by TianzhongSong** [comment]
@ldenoue  Batch norm may promote the performance, but i have not test it.
